npm-shrinkwrap.json

```diff
@@ -1,70 +1,72 @@
 {
   "name": "perseo",
-  "version": "0.4.0-next",
```

Comment:
Automatically generated. Some references have changed to include the URL. It does not seem a problem

NTC